iOSLearningApp/DetailView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DetailView.swift
+//  iOSLearningApp
+//
+//  Created by Chioma Amanda Mmegwa  on 09/10/2022.
+//
+
+import SwiftUI
+
+struct DetailView: View {
+    var body: some View {
+        VStack {
+            header
+            newsDetail
+        }
+        .padding()
+        .ignoresSafeArea()
+        .navigationBarHidden(true)
+    }
+    var header: some View {
+        NavigationBar(navigationTitle: "DETAIL", rightIcon1: "square.and.arrow.up", rightIcon2: "bookmark")
+            .padding(.top, 25)
+    }
+    var newsDetail: some View {
```

Comment:
no spacing, did you use swiftlint?

Reply:
Yes, I did. Does not throw up any errors. If i put any spaces, swiftlint flags it.